resqpy/olio/xml_et.py

```diff
@@ -124,6 +124,25 @@ def find_nested_tags(root, tag_list):
    return find_nested_tags(head, tag_list[1:])
 
 
+def find_nested_tags_cast(root, tag_list, dtype=None):
```

Comment:
Let's move function to resqpy.olio.xml_et

Reply:
It's already there